src/components/icon/utils.ts

```diff
@@ -1,11 +1,13 @@
-import { useTheme } from '@atb/modules/theme';
+import { useTheme, theme } from '@atb/modules/theme';
 
-export type SizeProps = 'small' | 'normal' | 'large' | 'x-large';
+export type SizeProps = keyof typeof theme.light.icon.size | 'x-large';
```

Comment:
`'x-large'` not in the theme?

Reply:
It's not in use in planner-web either, I see. And there's a TODO there that it should be in the design system at some point 😅

Reply:
Oh 😅 I'll just remove it, then!